Lesson21/todo-app/sahra/script.js

```diff
@@ -0,0 +1,39 @@
+document
+  .getElementById("todoForm")
+  .addEventListener("submit", function (event) {
+    event.preventDefault();});
+
+  const todoInputElement = document.getElementById("todoInput");
+  const toDoListElement = document.getElementById("toDoList");
+  function addItem(){
+    if(todoInputElement.value === ''){
```

Comment:
Make sure to trim input so that empty string is not added as a todo
```suggestion
    if(todoInputElement.value.trim() === ''){
```